plugins/@grouparoo/mailchimp/src/lib/shared/updateProfile.ts

```diff
@@ -130,8 +130,13 @@ export const updateProfile: UpdateProfileMethod = async ({
       tags: tagPayload,
     });
   } catch (error) {
-    // sometimes there are multiple errors that mailchimp puts in error.errors (which are json objects)
+    const errorMessage = error?.response?.body?.detail || "";
+    // in case of strict validation cases the error must be set to info level.
+    if (errorMessage.includes("looks fake or invalid")) {
+      error.errorLevel = "info";
+    }
     if (error.errors) {
+      // sometimes there are multiple errors that mailchimp puts in error.errors (which are json objects)
```

Comment:
Is it possible that in the case where there are more than one errors (`error.errors`), one of them could also be the `looks fake or invalid` message from the Mailchimp API?